docs/source/markdown/podman-container-restore.1.md

```diff
@@ -4,120 +4,131 @@
+#### **--all**, **-a**
+
+Restore all checkpointed *containers*.\
+The default is **false**.\
+*IMPORTANT: This OPTION does not need a container name or ID as input argument.*
+
 #### **--keep**, **-k**
 
-Keep all temporary log and statistics files created by CRIU during
+Keep all temporary log and statistics files created by `CRIU` during
 checkpointing as well as restoring. These files are not deleted if restoring
 fails for further debugging. If restoring succeeds these files are
 theoretically not needed, but if these files are needed Podman can keep the
 files for further analysis. This includes the checkpoint directory with all
 files created during checkpointing. The size required by the checkpoint
 directory is roughly the same as the amount of memory required by the
-processes in the checkpointed container.
-
-Without the **-k**, **--keep** option the checkpoint will be consumed and cannot be used
-again.
-
-#### **--all**, **-a**
-
-Restore all checkpointed containers.
+processes in the checkpointed *container*.\
+Without the **--keep**, **-k** option the checkpoint will be consumed and cannot be used again.\
+The default is **false**.
 
 #### **--latest**, **-l**
 
-Instead of providing the container name or ID, restore the last created container. (This option is not available with the remote Podman client)
-
-#### **--tcp-established**
-
-Restore a container with established TCP connections. If the checkpoint image
-contains established TCP connections, this option is required during restore.
-If the checkpoint image does not contain established TCP connections this
-option is ignored. Defaults to not restoring containers with established TCP
-connections.
-
-#### **--import**, **-i**
-
-Import a checkpoint tar.gz file, which was exported by Podman. This can be used
-to import a checkpointed container from another host. Do not specify a *container*
-argument when using this option.
-
-#### **--import-previous**
-
-Import a pre-checkpoint tar.gz file which was exported by Podman. This option
-must be used with **-i** or **--import**. It only works on runc 1.0-rc3 or higher.
-
-#### **--name**, **-n**
-
-This is only available in combination with **--import, -i**. If a container is restored
-from a checkpoint tar.gz file it is possible to rename it with **--name, -n**. This
-way it is possible to restore a container from a checkpoint multiple times with different
-names.
-
-If the **--name, -n** option is used, Podman will not attempt to assign the same IP
-address to the container it was using before checkpointing as each IP address can only
-be used once and the restored container will have another IP address. This also means
-that **--name, -n** cannot be used in combination with **--tcp-established**.
+Instead of providing the *container ID* or *name*, use the last created *container*. If other methods than Podman are used to run *containers* such as `CRI-O`, the last started *container* could be from either of those methods.\
```

Comment:
I think I like tool here instead of method.  Thoughts?

```suggestion
Instead of providing the *container ID* or *name*, use the last created *container*. If tools other than Podman are used to run *containers* such as `CRI-O`, the last started *container* could be from either tools.\
```

Reply:
Changed it.